application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/messages/internal/Messages.java

```diff
@@ -523,6 +523,7 @@
 	public static String VMOperationPage_PAGE_NAME;
 	public static String VMOperationPage_ROW_VM_OPERATIONS;
 	public static String VMOperationPage_TIMELINE_SELECTION;
+	public static String Flameview_UNCLASSIFIABLE_FRAME_DESC;
```

Comment:
Since it's only the flame view that uses this description, it could be in FlameView (it really should be, if it is to be named FlameView something). If we want the description to be generally available, it should be somewhere else. If we want it to be available to rules etc, it probably should live somewhere in core, near to where the definition of the unclassifiable frame is, e.g. in flightrecorder. One possible solution, would e.g. be to add a public messages bundle in the stacktrace package.

Reply:
thanks, corrected, I had the solution in stacktrace bundle before as we want to add similar tooltip to the stactrace view